assignments/lambda-classes.js

```diff
@@ -1 +1,122 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(personStats){
+        this.name = personStats.name;
+        this.age = personStats.age;
+        this.location = personStats.location;
+        this.gender = personStats.gender;
+    }
+
+    speak() {
+        return `Hello, my name is ${this.name} and I am from ${this.location}.`;
+    }
+}
+
+class Instructor extends Person {
+    constructor(InstructorStats){
+        super(InstructorStats);
+        this.specialty = InstructorStats.specialty;
+        this.favLangauge = InstructorStats.favLangauge;
+        this.catchPhrase = InstructorStats.catchPhrase;
+    }
+
+    demo(subject) {
+        return `Today we are learning about ${subject}`;
+    };
+
+    grade(student, subject) {
+        return `${student.name} received a perfect score on ${subject}.`;
+    };
+
+}
+
+class Student extends Person {
+    constructor(StudentStats) {
+        super(StudentStats);
+        this.previousBackground = StudentStats.previousBackground;
+        this.className = StudentStats.className;
+        this.favSubjects = StudentStats.favSubjects;
+    }
+
+    listSubjects() {
+        for(let i = 0; i < this.favSubjects.length; i++) {
+            console.log(this.favSubjects[i]);
+        };
+    };
+
+    PRAssignment(subject) {
+        return `${this.name} submitted a PR for ${subject}.` ;
+    };
+
+    sprintChallenge(subject) {
+        return `${this.name} has begune sprint challenge on ${subject}.`;
+    };
+}
+
+class ProjectManager extends Instructor {
+    constructor(PMStats) {
+        super(PMStats);
+        this.gradClassName = PMStats.gradClassName;
+        this.favInstructor = PMStats.favInstructor;
+    }
+
+    standUp(channel) {
+        return `${this.name} announces to ${channel}: ${channel} study time!`;
+    }
+
+    debugsCode(student, subject) {
+        return `${this.name} debugs ${student.name}'s code on ${subject}.`;
+    }
+}
+
+// Example Classes:
+
+const Jan = new Instructor({
```

Comment:
```suggestion
    const jan = ...
```

although this isn't a syntax rule, only the class itself will begin with a capital letter.  Since this is just a variable that is being assigned to the the instantiation of a new object from the class it is still best practice to not capitalize it.